### DISCUSSION TITLE
Introduction for new users.

### DISCUSSION THREAD

**Discussion by duyennguyen90** [FLOSIC/PublicRelease_2020#18] [General]
Hi everyone, 
It would be useful for new users if we have a short introduction on how to use the FLOSIC code.  Or some frequent issues when using FLOSIC code and how to deal with them. I am not sure if it is a useful idea. 
Best

**Comment by jacks1ka** [comment]
Hi Duyen-
  I am attaching the current version of the FLOSIC user manual.  We are beginning the process of updating the manual.  If you have suggestions, please let me know.
[FLOSIC_Tutorials_and_Documentation.pdf]
[FLOSIC_Tutorials_and_Documentation.pdf]
(https://github.com/FLOSIC/PublicRelease_2020/files/7138452/FLOSIC_Tutorials_and_Documentation.pdf)
